evaluate2.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#
+#  Copyright 2023 The original authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+if [ -z "$1" ]
+  then
+    echo "Usage: evaluate2.sh <fork name> (<fork name 2> ...)"
+    exit 1
+fi
+
+function check_command_installed {
+  if ! [ -x "$(command -v $1)" ]; then
+    echo "Error: $1 is not installed." >&2
+    exit 1
+  fi
+}
+check_command_installed java
+check_command_installed mvn
+check_command_installed hyperfine
+check_command_installed jq
+
+set -o xtrace
+
+java --version
+
+mvn -q clean verify
+
+rm -f measurements.txt
+ln -s measurements_1B.txt measurements.txt
+
+set +o xtrace
+
+echo ""
+TEMP_FILE=$(mktemp)
+OPTS="--warmup 1 --runs 5 --export-json $TEMP_FILE"
```

Comment:
Note: I configured 1 warmup run... this differs from `evaluate.sh` so we could always change this to 0